AstroAdapt.GraphQL/Mutation.cs

```diff
@@ -18,8 +17,8 @@ public class Mutation
         /// <returns>The saved solution.</returns>
         public async Task<SavedSolution> SaveSolutionAsync(
             Solution solution,
-            [Service]IAstroApp astroService) =>
-            await astroService.SaveSolutionAsync(solution);                    
+            IAstroApp astroService) =>
```

Comment:
I registered the `IAstroApp` as a common service in the `Program.cs` so you do no longer need to add the `ServiceAttribute`. This can unclutter your code.